### ISSUE TITLE
Crash when app name is in Chinese

### ISSUE THREAD

**Issue by guillaume-tgl** [helpscout/beacon-android-sdk-sample#57]
* Beacon version: 0.3.1
* Android OS version: Any
* Device make/model: Any

**Steps to reproduce**
1. Provide an app name in Chinese
2. Set the language of the device to Chinese
3. Try to open `BeaconActivity`

**Actual outcome**
It crashes with error:

```
java.lang.IllegalArgumentException: Unexpected char 0x7cbe at 0 in User-Agent value: 精靈掃描 Android/0.3.1 (samsung; SM-G960F; Android 9)
        at okhttp3.Headers$Builder.checkNameAndValue(Headers.java:352)
        at okhttp3.Headers$Builder.set(Headers.java:332)
        at okhttp3.Request$Builder.header(Request.java:184)
        at com.helpscout.beacon.internal.core.api.UserAgentInterceptor.intercept(UserAgentInterceptor.kt:29)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
        at okhttp3.RealCall$AsyncCall.execute(RealCall.java:147)
        at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:764)
```

**Expected outcome**
The header should probably be set to english language, rather than the locale of the device.

**Comment by scottyab** [comment]
Hi @guillaume-tgl thanks for raising this. It's not something we've come across and agree with your assessment we should sanitize the header values before sending to the server. I've logged this internally and we'll loop back with you here once it's fixed. 

I see you're still on 0.3.1, we released v1.0.3 last week and recommend upgrading to latest.

**Comment by guillaume-tgl** [comment]
Yes, thanks for reminding this :)

**Comment by guillaume-tgl** [comment]
Hi, any idea when this will be fixed? Our app cannot be used in Chinese until this is fixed.

**Comment by jotaemepereira** [comment]
@guillaume-tgl I do not have an exact date, but I estimate it's going to be this week. We released a new version with a fix for this to the QA team on Friday. After the QA validates the build, we will promote it to production. I will let you know as soon is up.

**Comment by jotaemepereira** [comment]
@guillaume-tgl We just released 1.0.5 with a fix for this crash, let us know how it goes!
